### ISSUE TITLE
Getting `$app` module not found error for component which is using `sveltkit` in REPL

### ISSUE THREAD

**Issue by thecodejack** [sveltejs/sites#262]
Can we add support for some of sveltekit internal API especially `$app` (https://kit.svelte.dev/docs#modules-$app-env) ?

**Comment by Conduitry** [comment]
Getting SvelteKit to work in the REPL is its own (quite large) project which we haven't begun to tackle yet, and in the meantime, I don't see much value in implementing various little pieces of it.

**Comment by bluwy** [comment]
SvelteKit's modules aren't independent in the sense that we can extract parts of it out. They are tightly integrated with the framework, so the most we can do is to shim it, which doesn't feel right. And implementing only parts of it would be confusing. Not to mention that the modules API can have breaking changes, which can break existing REPLs.

**Comment by thecodejack** [comment]
agreed. So what should be right way to handle those? Should there be an exposed node module as a shim?

**Comment by bluwy** [comment]
I think we should just not shim anything and keep the REPL vanilla. I'm not sure if it's worth the effort and instead could open a can of worms.

**Comment by thecodejack** [comment]
yea..not saying to add in REPL. was just thinking out loud if we can have a shim node module which will expose `$app` as a module available for import. That node module can be imported in REPL code wherever required.

**Comment by benmccann** [comment]
Related SvelteKit issue: https://github.com/sveltejs/kit/issues/1485
